src/components/entity/ha-chart-base.html

```diff
@@ -151,6 +151,21 @@
           value: false,
           readOnly: true,
         },
+        metas: {
+          type: Array,
+          value: () => [],
+        },
+        tooltip: {
+          type: Object,
+          value: () => ({
```

Comment:
why do you use arrow functions here and for metas?

Reply:
Objects are mutable data structures. Since we override properties of it, those properties would also be set on future instances of this web component.

If you define a function, it will be invoked to generate a default value, meaning each instance gets their own version of the object.